src/cards.csv

```diff
@@ -135,115 +135,111 @@
 "151","Scope change","Who negotiates scope changes?","g","3","","",""
-"150","Underestimate","If an item is larger than was estimated during sprint planning, what is most likely to happen?","g","2","","",""
+"150","Underestimate","What is likely to happen if we underestimate the size of an item during sprint planning?","g","2","","",""
 "149","Revenue","How much does the client expect to spend on Futurice before the end of the project?","g","1","","",""
-"148","Budget check","How often do we check how we are doing on budget?","g","3","","",""
+"148","Budget check","How often do we check the budget?","g","3","","",""
 "147","Timeline","How much budget do we expect to be used before we deliver the first working version?","g","1","","",""
-"146","Timeline","How much time is there reserved for bug fixing?","g","2","","",""
+"146","Timeline","How much time is reserved for bug fixing?","g","2","","",""
 "145","Timeline","When does the client expect the project to be released?","g","1","","",""
 "144","Responsibility","Whose ass on the client's is on the line if the project fails utterly?","g","3","","",""
 "143","Problem","What problem does this service try to solve?","a","1","","",""
-"142","Project aim","What is the aim of the client with this project?","a","1","Generally something like reduce costs, get more users, retain users, change business model...","",""
-"141","Why","What is the reason the client wants us to do the project?","a","1","","",""
+"142","Project aim","What is the client’s aim with this project?","a","1","Generally something like reduce costs, get more users, retain users, change business model...","",""
+"141","Why","Why does the client want Futurice to do this project?","a","1","","",""
 "140","Vision","What is the long-term vision of the service?","a","1","","",""
 "139","Release frequency","How frequent are production releases?","g","2","","",""
-"138","Product Owner","How long has the PO been working for the client organization?","g","3","","",""
-"137","SW buyer","Has the client a long track record of buying software?","a","1","","",""
-"136","Dependencies","Can we easily communicate with technical people of services we depend on?","g","2","","",""
-"135","PO time","How much time does the PO have available for the project?","g","3","","",""
-"134","Assets","How do we get client assets?","g","2","Think about texts, logo's brand material etc.","",""
+"138","Product Owner","How long has the Product Owner been working for the client organisation?","g","3","","",""
+"137","SW buyer","Does the client have a long track record of buying software?","a","1","","",""
+"136","Dependencies","Can we easily communicate with any 3rd party services we depend on?","g","2","","",""
+"135","PO time","How much time does the Product Owner have available for the project?","g","3","","",""
+"134","Assets","How can we access client assets like fonts, logos, brand material, research etc?","g","2","Think about texts, logo's brand material etc.","",""
```

Comment:
With this improvement the help text is not needed anymore

Reply:
👍